support/build/_util/include/manifest.py

```diff
@@ -23,7 +27,36 @@
 	"replace": json.loads(sys.argv[7]) if len(sys.argv) > 7 else {},
 	"provide": json.loads(sys.argv[8]) if len(sys.argv) > 8 else {},
 	"extra": json.loads(sys.argv[9]) if len(sys.argv) > 9 else {},
-	"time": datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S")
+	"time": os.getenv("NOW", datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S"))
 }
 
+# if it's a PHP manifest, we will generate full manifests for each shared extension into extra.shared; mkrepo.sh will then expand those into actual package declarations when it generates the repo
```

Comment:
Could you wrap these comments? I have to scroll sideways in the web UI to read them when reviewing otherwise, which makes reviews slower.